orchestrator/test_runner/src/bootstrapping.rs

```diff
@@ -132,9 +138,10 @@ pub async fn deploy_contracts(contact: &Contact) {
                 &format!("--eth-node={}", ETH_NODE.as_str()),
                 &format!("--eth-privkey={:#x}", *MINER_PRIVATE_KEY),
                 &format!("--contract={}", C[1]),
+                &format!("--contractERC721={}", C[2]),
                 "--test-mode=true",
             ])
-            .current_dir(C[2])
+            .current_dir(C[3])
```

Comment:
this looks duplicated from the other pr, please rebase off of main.

Reply:
Fixed